src/uct/ib/rc/verbs/rc_verbs_iface.c

```diff
@@ -262,6 +262,13 @@ static UCS_CLASS_INIT_FUNC(uct_rc_verbs_iface_t, uct_md_h tl_md,
     init_attr.cq_len[UCT_IB_DIR_TX]  = config->super.tx_cq_len;
     init_attr.seg_size               = config->super.super.super.seg_size;
 
+    status = uct_ib_iface_select_sl(&config->super.super.super, UCS_NO, 0, 0,
+                                    params->mode.device.dev_name,
+                                    &init_attr.sl);
+    if (status != UCS_OK) {
+        return status;
+    }
```

Comment:
why needed ? verbs transports do not support ooo_sl_mask
we should do it only for mlx5